src/components/graph/hotbar/Hotbar.tsx

```diff
@@ -202,4 +231,4 @@ const Hotbar: FC = () => {
   );
 };
 
-export default Hotbar;
+export default WithAuth(Hotbar);
```

Comment:
Maybe add a brief comment above this line to remind why you need WithAuth here as it may be a bit confusing.